### PR TITLE
Newsletter-110: translate into Chinese

### PR DESCRIPTION
**Pull request by bittomhan**

### CODE REVIEW

**Review by bitschmidty — CHANGES_REQUESTED** [review]
@bittomhan would you mind rebasing on master to resolve the conflicts with zh109

**Comment by bittomhan** [comment]
> @bittomhan would you mind rebasing on master to resolve the conflicts with zh109

I have update all links under `newsletter/zh/` path. 
Thank you for your check. This is also a chance for me to pratice `git`.

File: _posts/zh/newsletters/2020-08-12-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+{% include references.md %}
+{% include linkers/issues.md issues="18991,19620,3909,1499,19031,19606,192" %}
+[lnd 0.11.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.11.0-beta.rc2
+[news69 signcheck rpc]: /en/newsletters/2019/10/23/#c-lightning-3150
+[news95 ln atomicity]: /en/newsletters/2020/04/29/#new-attack-against-ln-payment-atomicity
+[myers anyprevout]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-August/018069.html
+[zmnscpxj reply]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-August/018071.html
+[corallo relay]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2020-August/018072.html
+[news108 wtxid]: /en/newsletters/2020/07/29/#bitcoin-core-18044
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2020-08-12-newsletter.md:88]:
```suggestion
[news108 wtxid]: /zh/newsletters/2020/07/29/#bitcoin-core-18044
```

File: _posts/zh/newsletters/2020-08-12-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+- [Bitcoin Core #19620][] 阻止 Bitcoin Core 重新下载试图花费非标准 UTXO 的未确认 segwit 交易。非标准 UTXO 是指使用当前不推荐的功能的 UTXO，例如在网络通过软分叉如 [taproot][topic taproot] 之前使用的 v1 segwit 输出。这有助于解决在 [Newsletter #108][news108 wtxid] 中描述的关于 taproot 激活的潜在问题：没有为软分叉升级的节点不会接受未确认的 taproot 交易，因此它们可能会一次又一次地下载并拒绝同样的未确认 taproot 交易。运行此补丁的节点不会出现这种情况，从而使该补丁的回溯移植成为[回溯移植 wtxid 中继功能][Bitcoin Core #19606]的一种替代方案，该功能也可以防止这种带宽浪费。
+
+- [C-Lightning #3909][] 更新了 `listpays` RPC，现在返回一个新的 `created_at` 字段，其中包含一个时间戳，指示首次部分支付的创建时间。
+
+- [Eclair #1499][] 添加了新的 `signmessage` 和 `verifymessage` 命令，分别使用您的 LN 节点的公钥签署消息并使用已知的节点公钥验证消息。测试表明，这些新命令与 [LND][LND #192] 和 [C-Lightning][news69 signcheck rpc] 中现有的消息签名和验证命令兼容，不同之处在于 Eclair 的签名以十六进制编码而不是 zbase32。
+
+{% include references.md %}
+{% include linkers/issues.md issues="18991,19620,3909,1499,19031,19606,192" %}
+[lnd 0.11.0-beta]: https://github.com/lightningnetwork/lnd/releases/tag/v0.11.0-beta.rc2
+[news69 signcheck rpc]: /en/newsletters/2019/10/23/#c-lightning-3150
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2020-08-12-newsletter.md:83]:
```suggestion
[news69 signcheck rpc]: /zh/newsletters/2019/10/23/#c-lightning-3150
```

File: _posts/zh/newsletters/2020-08-12-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+[news95 ln atomicity]: /en/newsletters/2020/04/29/#new-attack-against-ln-payment-atomicity
```
Comment (bitschmidty) [inline _posts/zh/newsletters/2020-08-12-newsletter.md:84]:
```suggestion
[news95 ln atomicity]: /zh/newsletters/2020/04/29/#new-attack-against-ln-payment-atomicity
```

**Review by bitschmidty — CHANGES_REQUESTED** [review]

**Review by bitschmidty — APPROVED** [review]

**Pull request merged**